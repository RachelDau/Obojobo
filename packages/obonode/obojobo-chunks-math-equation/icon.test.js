import React from 'react'
import renderer from 'react-test-renderer'

import Icon from './icon'

describe('MathEquation Icon', () => {
	test('Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
