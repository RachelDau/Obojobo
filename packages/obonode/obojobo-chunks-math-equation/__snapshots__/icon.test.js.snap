// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MathEquation Icon Icon 1`] = `
<svg
  viewBox="0 0 32 32"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    className="fill"
    d="M11.54,15.24a25.1,25.1,0,0,1-.64,7,2.85,2.85,0,0,1-.83,1.41,1.93,1.93,0,0,1-1.21.4,1.67,1.67,0,0,1-1.14-.35,1.76,1.76,0,0,1-.59-.82A4.78,4.78,0,0,1,7,21.76c0-.39,0-.78,0-1.17h.64a2.27,2.27,0,0,0,.24,1,.9.9,0,0,0,.8.56c.81,0,1.39-.72,1.73-2.12a25.65,25.65,0,0,0,.46-5.47v-.77H9.42a1.59,1.59,0,0,0-1.12.36,1.73,1.73,0,0,0-.53.92H7.13a9.12,9.12,0,0,1,.37-1.48,2.6,2.6,0,0,1,.63-1A2.84,2.84,0,0,1,9.35,12a10.59,10.59,0,0,1,2-.15c3,.05,5.61.11,7.82.2a1,1,0,0,0,.79-.26.85.85,0,0,0,.21-.45h.51a2.93,2.93,0,0,1-.54,1.95,1.9,1.9,0,0,1-1.48.49h-1.3v7.51c0,1.21.34,1.83,1.12,1.85a.91.91,0,0,0,.74-.37,2.47,2.47,0,0,0,.44-.78,9.17,9.17,0,0,0,.25-2h.63c0,.41,0,.85-.05,1.32a4.72,4.72,0,0,1-.38,1.33,2.3,2.3,0,0,1-.84,1,2.62,2.62,0,0,1-1.53.45,2.42,2.42,0,0,1-1.24-.28,2.46,2.46,0,0,1-.82-.7,3.64,3.64,0,0,1-.56-2V13.76H11.54Z"
  />
  <path
    className="fill"
    d="M22.2,13.23a34.94,34.94,0,0,0,2.67-3,3.17,3.17,0,0,0,.71-1.86,1.62,1.62,0,0,0-.43-1.21,1.43,1.43,0,0,0-1-.42,1.48,1.48,0,0,0-1.19.54,3.17,3.17,0,0,0-.49.9l-.23,0a2.84,2.84,0,0,1,.91-1.72A2.27,2.27,0,0,1,24.5,6,2,2,0,0,1,26,6.52a1.9,1.9,0,0,1,.56,1.4,2.84,2.84,0,0,1-.6,1.7c-.41.54-1.29,1.51-2.64,2.9h2.46a1.57,1.57,0,0,0,.73-.12,1.69,1.69,0,0,0,.47-.6l.14.06-.6,1.5H22.2Z"
  />
</svg>
`;
